src/js/modules/infragistics.ui.editors.js

```diff
@@ -3262,12 +3416,37 @@
 				```
 			*/
 			allowNullValue: false,
-			/* type="number" Gets/Sets delta-value which is used to increment or decrement value in editor on spin events. If value is set to negative value an exception is thrown. Non integer value is supported only for dataMode double and float.*/
+			/* type="number" Gets/Sets delta-value which is used to increment or decrement value in editor on spin events. If value is set to negative value an exception is thrown. Non integer value is supported only for dataMode double and float.
```

Comment:
what is spin events? I think the comment should explain that this happens when the spin-up and spin-down buttons are used. Also link to the option which enables the spin buttons.